test/extensions/transformers/aws_lambda/transformer_test.cc

```diff
@@ -3,13 +3,8 @@
 #include "source/extensions/transformers/aws_lambda/api_gateway_transformer.h"
 
 #include "test/mocks/http/mocks.h"
-#include "test/mocks/server/mocks.h"
-#include "test/test_common/utility.h"
 
-#include "fmt/format.h"
 #include "nlohmann/json.hpp"
-#include "gmock/gmock.h"
-#include "gtest/gtest.h"
```

Comment:
removed (along with the changes to the BUILD file) to improve compilation times for these tests

on my machine, this reduced the time to compile small changes to the already-cached suite from 80s to 60s